sensor/common/admissioncontroller/management_service.go

```diff
@@ -139,8 +139,8 @@ func (s *managementService) Communicate(stream sensor.AdmissionControlManagement
 }
 
 func (s *managementService) PolicyAlerts(_ context.Context, alerts *sensor.AdmissionControlAlerts) (*types.Empty, error) {
-	go s.alertHandler.ProcessAlerts(alerts)
```

Comment:
What are the consequences of making `PolicyAlerts` no longer start a goroutine?

Reply:
I would say none. `ProcessAlerts` spawns another goroutine, so we won’t block the call for much time. We could make the service aware of central’s connection state instead of the `alertHandler` but we’d have to make the service a SensorComponent. IMO I think it’s ok to keep it this way.